generate_command.sh

```diff
@@ -2,17 +2,22 @@
 
 source user_config.sh
 source fixed_config.sh
+source utils.sh
 
 this_dir=$(pwd)
+public_ip=$(discover_public_ip)
```

Comment:
change to private IP.

Reply:
The current version is working.

Reply:
Mmm ... I don't think it will work if access to public IP is blocked? 
Private IP is guaranteed. Take the case of McDonalds - this won't work I believe.

Reply:
But, if it is working - I think it's fine to keep it.